rkt/rm.go

```diff
@@ -48,39 +49,57 @@ func runRm(cmd *cobra.Command, args []string) (exit int) {
 			stderr("Unable to read UUID from file: %v", err)
 			return 1
 		}
-
-	case len(args) == 1 && flagUUIDFile == "":
-		podUUID, err = resolveUUID(args[0])
-		if err != nil {
-			stderr("Unable to resolve UUID: %v", err)
-			return 1
+		podUUIDs = append(podUUIDs, podUUID)
+
+	case len(args) > 0 && flagUUIDFile == "":
+		for _, uuid := range args {
+			podUUID, err := resolveUUID(uuid)
+			if err != nil {
+				stderr("Unable to resolve UUID: %v", err)
+			} else {
+				podUUIDs = append(podUUIDs, podUUID)
+			}
 		}
+
 	default:
 		cmd.Usage()
 		return 1
 	}
 
-	p, err := getPod(podUUID)
-	if err != nil {
-		stderr("Cannot get pod: %v", err)
-		return 1
+	ret := 0
+	for _, podUUID = range podUUIDs {
+		p, err := getPod(podUUID)
+		if err != nil {
+			ret = 1
+			stderr("Cannot get pod: %v", err)
+		}
+
+		if removePod(p) == 0 {
```

Comment:
Since the return value of removePod() isn't being returned directly out of here anymore, make it return a bool and switch this to
> if !removePod() {